pkg-manager/plugin-commands-installation/package.json

```diff
@@ -110,6 +110,9 @@
   "peerDependencies": {
     "@pnpm/logger": "^5.1.0"
   },
+  "optionalDependencies": {
+    "@napi-rs/xattr": "catalog:"
```

Comment:
you said in your issue that node.js natively supports setting the attribute, then why this library is needed?